docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Future Release
     * Enhancements
         * Add support for automatically inferring the ``EmailAddress`` logical
           type (:pr:`1047`)
+        * Add outlier detection and box plots to ``WoodworkColumnAccessor`` (:pr:`1048`)
```

Comment:
Should we be more specific about the dictionaries here? Especially for box plot, as this could be interpreted as adding a graphical plot instead of just the data.

Reply:
Thoughts on 

> Add ability to calculate outliers and the statistical info required for box and whisker plots to WoodworkColumnAccessor

Reply:
Sounds good!